api/soap/mantisconnect.wsdl

```diff
@@ -546,6 +550,16 @@
   <part name="issue" type="tns:IssueData" /></message>
 <message name="mc_issue_addResponse">
   <part name="return" type="xsd:integer" /></message>
+<message name="mc_issue_reportRequest">
+  <part name="username" type="xsd:string" />
```

Comment:
There can be several reporting-only accounts set up (one per project/app), and currently Mantis uses username (which has to be non-empty even with tokens) to determinate user id, against which it then checks token. So I don't see how this can be removed.